test/functorch/test_aotdispatch.py

```diff
@@ -2796,6 +2796,9 @@ def forward(self, x):
     xfail('sparse.sampled_addmm'),
     xfail('normal', 'number_mean'),  # TypeError: randn_like(): argument 'input' (position 1) must be Tensor, not float
     xfail('sparse.mm', 'reduce'),
+    xfail("nn.functional.max_unpool1d"),  # outputs are different
+    xfail("nn.functional.max_unpool2d"),  # outputs are different
+    xfail("nn.functional.max_unpool3d"),  # outputs are different
```

Comment:
wuh oh

Reply:
Indeed!

Reply:
Uh, the problem is just that the sample inputs we generate for max_unpool cause it to be nondeterministic. I can fix that

Reply:
Fixed the test to not generate inputs that cause it to be nondeterministic